e2e/k8s_test.go

```diff
@@ -156,3 +160,46 @@ func TestLargeDecodedVariablesSecretProvidedK8s(t *testing.T) {
 
 	testenv.Test(t, f.Feature())
 }
+
+*/
+
+func TestConjurSecretsNoPermissionK8s(t *testing.T) {
+	f := features.New("no permission to view conjur secrets").
+		// Replaces TEST_ID_12_no_conjur_secrets_permission
+		Assess("no permission to view conjur secrets in pod", func(ctx context.Context, t *testing.T, cfg *envconf.Config) context.Context {
+			// set environment variable in local environment
+			authenticatorID := os.Getenv("AUTHENTICATOR_ID")
+			appNamespaceName := os.Getenv("APP_NAMESPACE_NAME")
+
+			loginURI := fmt.Sprintf("host/conjur/authn-k8s/%s/apps/%s/service_account/%s-sa", authenticatorID, appNamespaceName, appNamespaceName)
+			fmt.Println(loginURI)
+
+			os.Setenv("CONJUR_AUTHN_LOGIN", loginURI)
+
+			// reload template with new login configuration
+			err := ReloadWithTemplate(cfg.Client(), K8sTemplate)
+			if err != nil {
+				fmt.Errorf("error reloading secrets provider: %s", err)
+			}
+
+			// get pod name and fetch logs from pod (we cannot run kubectl commands inside a container)
```

Comment:
this section appears to work properly but I am currently unable to test it with proper conjur authn configuration.

the logic of retrieving the pod name, checking the pod's logs, then checking the logs for expected error code is shown below